spec/order_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'order'
+
+describe Order do
+
+  context "#initialize" do
```

Comment:
I think context is used when testing behaviour state, and describe when testing a specific feature